data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -221,10 +226,46 @@ def write_low_to_file():
             )
             logger.info("Completed writing usa-low")
 
+        def write_esri_shapefile():
+            logger.info("Producing ESRI shapefiles")
+            # Note that esri shapefiles can't have super
+            # long column names, so we borrow from the
+            # tile names. We reconstruct the dictionary in case:
+            # 1. the set of columns changes, and
+            # 2. the length of column names changes
+            # Because these are shortened names, we should
+            # also print a key of these names.
+            codebook = {}
+            renaming_map = {}
+
+            # allows us to quickly rename / describe columns
+            reversed_tiles = {
+                short: long
+                for long, short in constants.TILES_SCORE_COLUMNS.items()
+            }
+            for column in self.geojson_score_usa_high.columns:
+                # take first 10 characters, max due to ESRI constraints
+                new_col = column[:10]
```

Comment:
we need to do this because ESRI shp can only have 10 character long column names -- so the idea here would be to create columns that are only 10 or fewer characters, and then also print out a codebook for ease of use -- since the end user might not understand `IALHSEF` as a field. So that's what this does.

Reply:
Interesting